### DISCUSSION TITLE
Player Indicators of Others (all players) only turning on for PVP scenarios

### DISCUSSION THREAD

**Discussion by BidMaBiza** [runelite/runelite#15200] [Ideas]
give us the option to have Player Indicators for Other players only toggled on within a dangerous PVP scenario (wildy/pvp worlds/pvp minigames)

if its allowed and or possible with level display after their name and if possible also able to toggle only for players within attack range (people that can attack you or vice versa)

^the second suggestion is literally QOL  since 60% of PVP in osrs is rightclicking people to see if you can attack them... seems kinda dumb to me
am i wrong?

shoudnt be considered OVERPOWERED but a feature of an evolving game in 2022

**Comment by Castro3halo** [comment]
100% agree hope this gets looked at, very annoying having to manually turn on when wanting to see names in wildy vs being overwhelmed by names at the GE
